gcs.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2018 Mikael Rapp, github.com/zatte
```

Comment:
Om det ska mergas in i spf13/afero, så vill han kanske inte att du petar in din egen Copyright ovan.

Reply:
Tänker han får själv avgöra ifall apache licence ger ju hon alla rättigheter han behöver samt tror det finns andra contributions med liknande wordings i repot redan